### PR TITLE
LPS-183675 & LPS-183541 [PRM] Update MDF Sales Page accordingly to Production Environment

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json [hunk 1]
```diff
@@ -342,7 +497,8 @@
 									{
 										"definition": {
 											"fragment": {
-												"key": "button-welcome-banner"
+												"key": "button-welcome-banner",
+												"siteKey": "PRM Testing Sales2"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json:501]:
```suggestion
												"siteKey": "PRM Testing Sales2"
```

Add the replace

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json [hunk 2]
```diff
@@ -396,1522 +552,135 @@
...
-																			}
-																		],
-																		"type": "Section"
+																				{
+																					"fieldName": "privatePage",
+																					"fieldValue": "false"
+																				},
+																				{
+																					"fieldName": "siteKey",
+																					"fieldValue": "PRM Testing Sales2"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json:573]:
```suggestion
																					"fieldValue": "PRM Testing Sales2"
```

replace here too

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json [hunk 3]
```diff
@@ -396,1522 +552,135 @@
...
-																],
-																"type": "Column"
+																}
 															}
-														],
-														"type": "Row"
+														},
+														"text": {
+															"value_i18n": {
+																"en_US": "Register New Deal"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json:583]:
```suggestion
																"en_US": "Register New Deal"
```

add another languages

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json [hunk 4]
```diff
@@ -3622,7 +2402,7 @@
 																								"key": "clay-icon-link"
 																							},
 																							"fragmentConfig": {
-																								"iconLink": "",
+																								"iconLink": "https://www.liferay.com/pt/home",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json:2405]:
```suggestion
																								"iconLink": "https://www.liferay.com/",
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by claricebrasil** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:stop

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:test:relevant

**Comment by claricebrasil** [comment]
ci:forward

**Pull request closed without merging**